src/dbnode/generated/proto/annotation/annotation.proto

```diff
@@ -3,11 +3,17 @@ syntax = "proto3";
 package annotation;
 
 message Payload {
-    MetricType metric_type   = 1;
+    SourceFormat source_format = 3;
+
+    // Used when source_format == PROMETHEUS
+    PrometheusType prom_type = 1;
     bool handle_value_resets = 2;
+
+    // Used when source_format == GRAPHITE
+    GraphiteType graphite_type = 4;
 }
 
-enum MetricType {
+enum PrometheusType {
```

Comment:
nit: As discussed perhaps let's call this `OpenMetricsMetricFamily` given DB does not need to distinguish between Prometheus type vs OpenMetrics type.

Reply:
Naming it slightly differently than suggested (`OpenMetricsFamilyType`).